backend/src/main/java/io/debezium/configserver/rest/client/jolokia/JolokiaClient.java

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright Debezium Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.debezium.configserver.rest.client.jolokia;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import java.util.stream.Collectors;
+import java.net.URI;
+import java.net.URISyntaxException;
+
+import javax.management.MalformedObjectNameException;
+
+import io.debezium.configserver.model.JolokiaURIsList;
+import io.smallrye.config.SmallRyeConfig;
+import jakarta.enterprise.context.ApplicationScoped;
+import jakarta.inject.Inject;
+
+import org.eclipse.microprofile.config.ConfigProvider;
+import org.eclipse.microprofile.config.inject.ConfigProperty;
+import org.jolokia.client.BasicAuthenticator;
+import org.jolokia.client.J4pClient;
+import org.jolokia.client.exception.J4pException;
+import org.jolokia.client.request.J4pReadRequest;
+import org.jolokia.client.request.J4pReadResponse;
+
+@ApplicationScoped
+public class JolokiaClient {
+    public static final int DEFAULT_JOLOKIA_PORT = 8778;
+    public static final String PROPERTY_JOLOKIA_CLIENT_URI = "jolokia.client.uri";
+    private static JolokiaURIsList jolokiaURIsList;
```

Comment:
Why are we not using List<URI> here instead of wrapping ArrayList with a custom data type?